src/modules/booking/controllers/booking-controller.js

```diff
@@ -215,102 +130,57 @@ class BookingController {
     }
   }
 
-  static async rejectBooking(req, res) {
+  //
+  static async searchBooking(req, res) {
     try {
```

Comment:
@tracy8 Please add search functionality using all this filters as mentioned in the story

<img width="354" alt="image" src="https://user-images.githubusercontent.com/60257249/230077404-636096e1-e00b-4818-99ea-6d8ed4ba99cb.png">